tasks/release.py

```diff
@@ -1378,3 +1382,82 @@ def check_omnibus_branches(_):
         if omnibus_software_version != 'master' and not version_re.match(omnibus_software_version):
             raise Exit(code=1, message=f'omnibus-software version [{omnibus_software_version}] is not mergeable')
     return True
+
+
+@task
+def update_build_links(_ctx, new_version):
+    """
+    Updates Agent release candidates build links on https://datadoghq.atlassian.net/wiki/spaces/agent/pages/2889876360/Build+links
+
+    new_version - should be given as an Agent 7 RC version, ie. '7.50.0-rc.1' format.
+
+    Notes:
+    Attlasian credentials are required to be available as ATLASSIAN_USERNAME and ATLASSIAN_PASSWORD as environment variables.
+    ATLASSIAN_USERNAME is typically an email address.
+    ATLASSIAN_PASSWORD is a token. See: https://id.atlassian.com/manage-profile/security/api-tokens
+    """
+    from atlassian import Confluence
```

Comment:
You might want to put this module in a requirements.txt file (eg `tasks/libs/requirements-github.txt`)
We might think about something that could warn about required packages for invoke task, as we have so many of them and some module are quite specific, like this one